src/canister_tests/src/tests.rs

```diff
@@ -24,23 +28,27 @@ fn ii_upgrade_retains_anchors() {
     let env = StateMachine::new();
     let canister_id = framework::install_ii_canister(&env, framework::II_WASM_PREVIOUS.clone());
     let challenge = api::create_challenge(&env, canister_id);
-    let user_number = match api::register(
-        &env,
+    let challenge_attempt = types::ChallengeAttempt {
```

Comment:
nit: unclear why this moved , was it for clarity? should we keep it consistent with the other test and inline as well?

Reply:
Hm, I need to be more careful with IDE powered refactorings

-> reverted.